code_schemes/imaqal_special_show.json

```diff
@@ -3,6 +3,78 @@
   "Name": "Imaqal Special Show",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "StringValue": "ali",
+      "DisplayText": "ali",
+      "VisibleInCoda": true,
+      "NumericValue": 0,
+      "CodeID": "code-6c88bf5a",
+      "CodeType": "Normal"
+    },
+    {
+      "NumericValue": 1,
+      "CodeID": "code-9c9847a6",
+      "CodeType": "Normal",
+      "StringValue": "ismail",
+      "DisplayText": "ismail",
+      "VisibleInCoda": true
+    },
+    {
+      "NumericValue": 2,
+      "CodeID": "code-4cd72bce",
+      "CodeType": "Normal",
+      "StringValue": "mumina",
+      "DisplayText": "mumina",
+      "VisibleInCoda": true
+    },
+    {
+      "NumericValue": 3,
+      "CodeID": "code-10ad4f77",
+      "CodeType": "Normal",
+      "StringValue": "positive_remarks_about_drama",
+      "DisplayText": "positive remarks about drama",
+      "VisibleInCoda": true
+    },
+    {
+      "StringValue": "bilan",
+      "DisplayText": "bilan",
+      "VisibleInCoda": true,
+      "NumericValue": 4,
+      "CodeID": "code-ddf04a8d",
+      "CodeType": "Normal"
+    },
+    {
+      "DisplayText": "jamila",
+      "VisibleInCoda": true,
+      "NumericValue": 5,
+      "CodeID": "code-84843eac",
+      "CodeType": "Normal",
+      "StringValue": "jamila"
+    },
+    {
+      "CodeType": "Normal",
+      "StringValue": "all_characters",
+      "DisplayText": "all characters",
+      "VisibleInCoda": true,
+      "NumericValue": 6,
+      "CodeID": "code-4bdddf48"
+    },
+    {
+      "StringValue": "negative_perception",
+      "DisplayText": "negative perception",
```

Comment:
Is this of the show (i.e. the "opposite" code for "positive remarks about drama") or is it referring to something else?

Reply:
chatting with RDA this is the opposite of positive remarks about drama. example response Reading the Quran is better than listening to your drama things. The question was asking which drama characters did you like in the recent drama shows.

Reply:
We do have the option to change this to negative remarks about drama or keep it as is RDA is happy with either

Reply:
I think "negative remarks about drama" is clearer, but it's really up to RDA, and what makes most sense for them - they're the experts here. Also, if labelling has started, then it's what the RAs interpreted "negative perception" to mean, and if we have to be confident that exactly align switch "negative remarks about drama" before changing.

Reply:
I agree will keep it as is to maintain sync with what RDA first interpreted and coded as.